### ISSUE TITLE
Voxel GI Sky Indirect Creates Banding Artifacts

### ISSUE THREAD

**Issue by WickedInsignia** [godotengine/godot#84746]
### Godot version

4.5 Beta 5

### System information

Windows 11, Nvidia RTX4070ti, AMD Ryzen7700x

### Issue description

Voxel GI creates extreme banding artifacts from sky indirect contribution when "interior" is unticked.
This is noticeable on perfectly matte materials and very noticeable on reflective materials. The effect is exacerbated by low domain subdiv levels but not entirely resolved by higher ones. This may be one of the main culprits behind issue #50842.

When sky indirect contribution is allowed, noticeable banding occurs on all materials and is most noticeable in shadowed areas on flat non-textural surfaces at normal to high exposures. This creates problems for low-poly and minimalist styles:
![VoxelGISkyBanding01](https://github.com/godotengine/godot/assets/70243134/939eeb7f-fa98-489e-950a-68a5d2dbde42)

When the probe is set to "inside" mode, the banding is not present (exposure increased to account for lack of available light):
![VoxelGISkyBanding02](https://github.com/godotengine/godot/assets/70243134/62415b3a-d08b-48f9-9b92-c3a6e706e9de)

This effect is very noticeable on reflective surfaces, with severe banding appearing on a range of roughness settings. Here is a nonmetallic material with minimum roughness:
![VoxelGISkyBanding04](https://github.com/godotengine/godot/assets/70243134/84de04fa-1505-4e06-a604-63fe1cc8f173)
![VoxelGISkyBanding03](https://github.com/godotengine/godot/assets/70243134/1ef1b63c-5171-4d2e-9afb-ed875f802d29)

These artifacts are creating some problems that are difficult to work around in my own projects, where they present very noticeably. Top image is exterior mode, bottom image is interior mode:
![VoxelGISkyBanding05](https://github.com/godotengine/godot/assets/70243134/f5fcebf8-3e13-4d27-a9ad-b77e635fc05a)
![VoxelGISkyBanding06](https://github.com/godotengine/godot/assets/70243134/63236729-6941-4d99-950d-773e496ad85d)
![VoxelGISkyBanding07](https://github.com/godotengine/godot/assets/70243134/3ef2878a-ef3f-42ce-896b-f5dd00337b13)
![VoxelGISkyBanding08](https://github.com/godotengine/godot/assets/70243134/ab09cca4-f118-4e9d-964e-72245ea47667)

### Steps to reproduce

Use VoxelGI in exterior mode on flat non-textural materials. Higher exposures in shadowed areas show the artifacts more readily.

### Minimal reproduction project

[VoxelGI Sky Banding.zip](https://github.com/godotengine/godot/files/13325141/VoxelGI.Sky.Banding.zip)

**Comment by Calinou** [comment]
Do these artifacts become less noticeable when setting the VoxelGI quality to **High** in the project settings?

**Comment by WickedInsignia** [comment]
> Do these artifacts become less noticeable when setting the VoxelGI quality to **High** in the project settings?

No, they become more noticeable since they occur at a higher frequency.

**Comment by jams3223** [comment]
Has this issue been fixed?

**Comment by Calinou** [comment]
> Has this issue been fixed?

I can still reproduce it as of 4.3.beta3. Note that reducing **Dynamic Range** to 1 in the VoxelGIData doesn't fix the issue. Adjusting bias or normal bias also doesn't help.

That said, this issue may be considered a duplicate of https://github.com/godotengine/godot/issues/50842.

**Comment by jams3223** [comment]
> > Has this issue been fixed?
> 
> I can still reproduce it as of 4.3.beta3. Note that reducing **Dynamic Range** to 1 in the VoxelGIData doesn't fix the issue. Adjusting bias or normal bias also doesn't help.
> 
> That said, this issue may be considered a duplicate of #50842 since they are due to the same cause (a bug or limitation in VoxelGI's voxel cone tracing for drawing reflections).

Hope Juan fixes them.
